src/js/components/List/List.js

```diff
@@ -499,6 +500,7 @@ const List = React.forwardRef(
                     {...boxProps}
                     {...clickProps}
                     {...orderProps}
+                    {...generalItemProps}
```

Comment:
If I specify both `itemProps` and `generalItemProps` but they have a conflict (e.g. border, background, pad), which styling wins? Which should win?

Reply:
good question..
I guess if someone is wanting to apply `itemProps` they are going to want it to a specific index so wondering if that should win.. I could see someone setting `background: 'background'` for the whole list then doing something like `itemProps: { index 1: background: 'contrast'}`

Reply:
Sounds good I’ll look into that!